lib/libutee/include/tee_api.h

```diff
@@ -33,6 +33,9 @@
 #include <compiler.h>
 #include <tee_api_defines.h>
 #include <tee_api_types.h>
+#if defined(CFG_TEE_PANIC_DEBUG)
```

Comment:
If you want a new flag, I would vote for something less specific, as for example `CFG_TEE_TA_DEBUG`. Having too many flags is a killer in my opinion. So we should create them wit care.